src/components/i18n/Composition.js

```diff
@@ -0,0 +1,131 @@
+/*
+ * Utility class for the FormattedCompMessage component.
+ */
+import React from 'react';
+import * as MA from 'message-accumulator';
+import * as N from 'ilib-tree-node';
+import { JSTYPE_BOOLEAN, JSTYPE_NUMBER, JSTYPE_OBJECT, JSTYPE_STRING } from '../../constants';
+
+const MessageAccumulator = MA.default;
```

Comment:
message-accumulator and ilib-tree-node are both ES5 CommonJS modules, so they can't be imported normally. I can perhaps use "require" instead of import to avoid this...